### ISSUE TITLE
Japanese Translation is incorrect (confusing) in Email router document

### ISSUE THREAD

**Issue by damedom** [MicrosoftDocs/feedback#602]
Japanese Translation is incorrect (confusing) in "Important" section.

Title : Integrate (synchronize) your email system
URL(en) : https://docs.microsoft.com/en-us/dynamics365/customer-engagement/admin/integrate-synchronize-your-email-system
URL(ja) : https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/admin/integrate-synchronize-your-email-system

English Version : 
> Important
> In previous versions of Dynamics CRM, you could also use the Email Router to synchronize records. The Email Router has been **deprecated** as of the Dynamics 365 (online), version 9.0. We strongly recommend that you migrate all email routing functionality to use server-side synchronization.

Japanese Version : 
> 旧バージョンの Dynamics CRM では、レコードの同期に Email Router も使用できました。 Email Router は Dynamics 365 (オンライン) バージョン 9.0 で**廃止されました**。 すべての電子メールのルーティング機能をサーバー側同期を使用するよう移行することをお勧めします。

## Issue Details
The expression **"廃止されました"** means "has been discontinued". People who read this article may believe that the feature is already deleted.  But actually, Email router functionality hasn't been deleteed in V9.0 

Title : Dynamics 365 (online), version 9.0 Email Router
URL :  https://www.microsoft.com/en-us/download/details.aspx?id=56974

This should be "廃止予定です" in Japanese which means "to be deleted soon" or "planned to be deleted soon". So it would be better to write like this:
> 旧バージョンの Dynamics CRM では、レコードの同期に Email Router も使用できました。 Email Router は Dynamics 365 (オンライン) バージョン 9.0 で**廃止予定です**。 すべての電子メールのルーティング機能をサーバー側同期を使用するよう移行することをお勧めします。

## Similar Issue
There's a same mistranslation here:

Title : Migrate settings from the Email Router to server-side synchronization
URL(en) : https://docs.microsoft.com/en-us/dynamics365/customer-engagement/admin/migrate-settings-email-router-server-side-synchronization
URL(ja) : https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/admin/migrate-settings-email-router-server-side-synchronization

English Version : 
> Note
> The Microsoft Dynamics CRM Email Router has been deprecated and was removed in version 9.0 of Dynamics 365. To prepare for this change, we strongly recommend that you migrate all email routing functionality to use the server-side synchronization feature.
> 

Japanese Version : 
> 注意
> Microsoft Dynamics CRM Email Router は**廃止され**、Dynamics 365 のバージョン 9.0 で**削除されました**。 この変更に対応するため、すべての電子メールのルーティング機能がサーバー側同期機能を使用するよう移行することをお勧めします。

The expression **"削除されました"** means "has been deleted". It would be better to write like this:
> 注意
> Microsoft Dynamics CRM Email Router は**廃止予定であり**、Dynamics 365 のバージョン 9.0 **以降** で**削除されることになっています**。 この変更に対応するため、すべての電子メールのルーティング機能がサーバー側同期機能を使用するよう移行することをお勧めします。

**Comment by yumetodo** [comment]
![image](https://user-images.githubusercontent.com/10869046/44017372-ff64448e-9f12-11e8-8895-d483ab576f85.png)

**Comment by Powerhelmsman** [comment]
@nokura @SonjaSaltzman Can you fix it?

**Comment by SonjaSaltzman** [comment]
This content is from @krisre / @vinceno  - could you please follow up?

**Comment by vinceno** [comment]
@mhogan73 @SonjaSaltzman - thx Sonja., bring in Marina in Kris' absence.

**Comment by mhogan73** [comment]
Thx @yumetodo we will look into this issue and update you on when we can address this. Thx

**Comment by SonjaSaltzman** [comment]
@mhogan73 can you please comment and close this if it has been addressed?

**Comment by SonjaSaltzman** [comment]
Also cc: @krisre

**Comment by mhogan73** [comment]
This has been addressed and is now published live.

**Comment by yumetodo** [comment]
Wait, wait, wait, wait.

I watch below now:

- https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/admin/integrate-synchronize-your-email-system
- https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/admin/migrate-settings-email-router-server-side-synchronization

It's seems that this issue is not resolved.

**Comment by mhogan73** [comment]
I will recheck the issue asap and update the bug.

**Comment by mhogan73** [comment]
Hi @yumetodo @damedom 
[Japanese issues.docx](https://github.com/MicrosoftDocs/feedback/files/2597726/Japanese.issues.docx)
- Apologies for incorrect information, there was confusion with another bug. 

Please see the document attached with feedback from our localization suppliers on the issues logged.

**Comment by yumetodo** [comment]
I quote the feedback from Localization Suppliers for discussion.

>The current translation is correct and follows the source, which says the feature "has been deprecated and was removed in version 9.0". However, it´s possible the reviewer knows this is not the case yet and wants the target to mean that this removal hasn´t already happened, therefore prefers to translate Japanese as ""planned to be deleted soon". We don´t recommend implementing this part, but we can change "deprecated" with the approved TS translation as in Issue 1. Please notice that this is Basic content, therefore while we haven´t followed TS, there is no possibility of the user misunderstanding the original concept re deprecated, the current translation is also widely-used and understood to be a synonym. 

---

>The current translation is correct and follows the source

**NOT CORRECT**.

It is already pointed by @damedom that why the current translation is not correct. @DaisukeEda also ponited out same problem at #881 and #882. However, Localization Suppliers does not understand.

So, I point out again.  `廃止された` means `already removed`. `削除予定` means `remove soon`.

>wants the target to mean that this removal hasn´t already happened, therefore prefers to translate Japanese as ""planned to be deleted soon".

Wait, wait, wait, wait.

Localization Suppliers are already decided that `"already removed"` and `"remove soon"` is same meaning (of course that is not correct), isn't it?

It is surprising that Localization Suppliers is planning to change the translation to `まもなく削除される予定`(`"planned to be deleted soon"`).

It's may be related issue that [Bing translate](https://www.bing.com/translator) says that `deprecated` means `廃止`. If Localization Suppliers use Bing translate to understand this issue, DO NOT USE Bing translate. Bing translate is broken, and we should deprecate to use Bing translate.

To avoid misunderstanding, I quote wiktionary(from  03:28, 25 July 2018 version):
https://en.wiktionary.org/wiki/deprecated

> (computing) Obsolescent; said of a construct in a computing language considered old, and planned to be phased out, but still available for use. 

Can you see "still available for use"? That is exactly this issue pointed out.

By the way, `まもなく削除される予定`(`"planned to be deleted soon"`) is too wordy translation(I think `非推奨` is better).

---

Please re-open this issue, [search `deprecated` on wiktionary](https://en.wiktionary.org/wiki/deprecated).

**Comment by mhogan73** [comment]
Hi @yumetodo I am discussing this issue again with our localization suppliers. Pls note they do not use Bing Translate as part of the translation process. The office here is closed today and tomorrow so I will update you on Monday. Thx

**Comment by mhogan73** [comment]
I have reviewed this issue again with the localization suppliers and some internal Japanese speakers and they have all agreed that the translation online is correct. The Organization Data Service was deprecated in CRM 2016 (version 8.x) and this has been reflected in the translation. 

Please note the current translation is following the source correctly.  " 廃止され" means "has been deprecated" , while the suggested translation of "廃止予定となり​” means "planned to be deprecated".

**Comment by blackwingcat** [comment]
Hi, mhogan73

I think you misunderstand "廃止"

"廃止" means "deleted" in Japan.

"廃止" means "you can not use function."
"廃止予定"   means "you can use function  but worse"
"非推奨" means "you can use function but worse"
"非推奨予定"   means  "it is official  function now"



"has been deprecated" means "has been not recommended" = "非推奨になった"
"planned to be deprecated " means "planned to be not recommended" = "非推奨予定"


for example 
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.5_release_notes/chap-red_hat_enterprise_linux-7.5_release_notes-deprecated_functionality

Python 2 has been deprecated  "Python 2 は非推奨になった"
"We can still use Python 2"


Sincerely.

**Comment by yumetodo** [comment]
from #881 
> **Link to article:**
> Title: Organization data service
> URL(en) : https://docs.microsoft.com/en-us/dynamics365/customer-engagement/developer/developer-resources-page#organization-data-service
> URL(jp) : https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/developer/developer-resources-page#organization-data-service
> 
> English Version :
> The Organization Data service has been deprecated and replaced by the Web API. For more information about the Web API see Use the Dynamics 365 Web API.
> 
> Japanese Version :
> 組織のサービスは、廃止され、Web API に置き換えられました。 Web API の詳細については、Dynamics 365 Web API の使用 を参照してください。
> 
> **Problem:**
> The expression " 廃止され" means "has been removed". People who read this article believe that the feature is already removed. "Deprecated" should transrate to "廃止予定" in Japanese. So that please change it as below.
> 
> 組織のサービスは、廃止予定となり、Web API に置き換えられました。 Web API の詳細については、Dynamics 365 Web API の使用 を参照してください。

from #882 

> **Link to article:**
> Title: Organization data service
> URL(en) : https://docs.microsoft.com/en-us/dynamics365/customer-engagement/developer/use-microsoft-dynamics-365-web-services#organization-data-service
> URL(jp) : https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/developer/use-microsoft-dynamics-365-web-services#organization-data-service
> 
> English Version :
> While this service will continue to be available as-is for years to come, we are deprecating it with Dynamics 365 (online).
> 
> Japanese Version :
> このサービスは、Dynamics 365 (online) を廃止する予定の年までは引き続き利用可能となります。
> 
> **Problem:**
> The transration of "we are deprecating it with Dynamics 365 (online)." is definitely incorrect.
> "このサービスは、Dynamics 365 (online) を廃止する予定の年までは引き続き利用可能となります。"
> means that we can use it until the year when Dynamics 365 (online) is end of service. So please correct as below.
> 
> The expression " 廃止され" means "has been removed". People who read this article believe that the feature is already removed. "Deprecated" should transrate to "廃止予定" in Japanese. So that please change it as below.
> 
> このサービスは、Dynamics 365 (online) にて廃止予定となっておりますが、利用可能となります。

**Comment by mhogan73** [comment]
We are revisiting this translation issue reported and hope to update this issue and also #881 and #882 tomorrow with a resolution. Thx.

**Comment by mhogan73** [comment]
@yumetodo We are making a change to the topic, this should be published live soon. I will update you when the topic is Live. Thanks.

**Comment by mhogan73** [comment]
@yumetodo Please note the updated topic should now be Live, can you please review ? Thanks. 

https://docs.microsoft.com/ja-jp/dynamics365/customer-engagement/developer/developer-resources-page#organization-data-service

**Comment by yumetodo** [comment]
@mhogan73 Perfect! Thank you.
